src/components/HeadlineArea/README.md

```diff
@@ -0,0 +1,26 @@
+# HeadlineArea
+
+```tsx
+import { HeadlineArea } from 'smarthr-ui'
+
+<HeadlineArea
+  heading={{ heading: 'HeadlineArea', tag: 'h1' }}
+  description="Description message."
+  className="className"
+/>
+```
+
+## props
+
+| Name        | Required | Type          | DefaultValue                  | Description                                                    |
+| ----------- | -------- | ------------- | ----------------------------- | ---------------------------------------------------------------|
+| heading     | true     | **object**    | [reference Heading](#Heading) | The Content of the Headline Area [reference Heading](#Heading) |
+| description | false    | **ReactNode** | undefined                     | description message.                                           |
+| className   | false    | **string**    | ''                            | `className` of component.                                      |
```

Comment:
It would be better if `required` is written in this `✓` or `-` style😀

![image](https://user-images.githubusercontent.com/14817308/86450141-0c54ad80-bd54-11ea-8be9-658a8847eb23.png)

Reply:
Ah, I've just noticed that it is not consistent in this repo😭
You can ignore this comment since `required` fields should be fixed thoroughly in another pull request👌

Reply:
Exactly, I issued this problem to our backlog :memo: